_posts/2021-09-23-hacktoberfest.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: Hacktoberfest 2021
+layout: post
+---
+
+Are you ready for Hacktoberfest 2021?  Void Linux is!  We're
+excited to be participating for our 5th year.  Contributions that help to
+address our out-of-date packages queue are especially welcome.
+These are all great ways to dip your feet into the world of Linux
```

Comment:
"these are all great ways" should be "this is a great way", I think. You only mentioned one way to contribute, above.